ably.d.ts

```diff
@@ -2516,9 +2516,8 @@ export interface PresenceMessageStatic {
    * Initialises a `PresenceMessage` from a `PresenceMessage`-like object.
    *
    * @param values - The values to intialise the `PresenceMessage` from.
-   * @param stringifyAction - Whether to convert the `action` field from a number to a string.
```

Comment:
Is removing this not a breaking public API change?

Reply:
Technically, yes.

But this is an parameter that is only ever used by the library to decode presence messages sent over a websocket from ably. A user of the library would never have any reason to use it. Indeed a user of the library would never have any reason to know or care that we use numerical encoding in our internal wire protocol. It should never have been exposed in the SDK public API in the first place.

(This method appears to only have been added to support a very niche usecase of sending extras in a presence event, so is not widely used. Possibly not at all outside of the spaces sdk.)